invenio_communities/assets/semantic-ui/js/invenio_communities/requests/index.js

```diff
@@ -76,24 +76,24 @@ const RequestCancelModalTriggerWithConfig = parametrize(RequestCancelModalTrigge
 });
 
 const CommunitySubmission = () => (
-  <LabelTypeSubmission className="primary" size="small" />
+  <LabelTypeSubmission className="primary" size="tiny" />
 );
 
 const CommunityInvitation = () => (
-  <LabelTypeInvitation className="primary" size="small" />
+  <LabelTypeInvitation className="primary" size="tiny" />
 );
 
-const Submitted = () => <LabelStatusSubmit className="primary" size="small" />;
+const Submitted = () => <LabelStatusSubmit className="primary" size="tiny" />;
```

Comment:
I would be a bit worried that it won't be readable anymore

Reply:
IMHO it's readable. It would be the same size as we have for the uploads page. In that same view we have some smaller text which I think is readable. However, if we want to highlight these labels a bit more, then I think we should increase the size for **all** the labels and find an alternative solution to center the icon on the left. 

Here an example of the uploads page with the **tiny** size .
![Screenshot from 2023-01-17 09-43-00](https://user-images.githubusercontent.com/15194802/212851303-0774b3bf-a47e-4e87-85b9-aa883d835659.png)

Reply:
I think you might be biased by the font changes in your PC/browser. IMO it is not readable, at least not on my screen.
Sacrificing readability and user experience, because of icon misalignment is not very well justified.

Reply:
Sorry I've not explained my self well. 
I just tried to align the icons on the entire website, as I mentioned in my previos comment the upload page is using the size tiny to display the tags while the requests page is using the size small. Since the alignment for the tiny labels is correct and we were using them already I thought it would make sense to harmonize them across the entire site. If you think that this size is not readable we should revise and update the uploads page because is currently using it (see [here an example](https://github.com/inveniosoftware/invenio-app-rdm/blob/master/invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/user_dashboard/uploads_items/ComputerTabletUploadsItem.js#L52) of it)